modules/vaos/app/services/vaos/appointment_service.rb

```diff
@@ -42,12 +42,14 @@ def put_cancel_appointment(request_object_body)
     private
 
     def deserialized_appointments(json_hash, type)
-      if type == 'va'
-        json_hash.dig(:data, :appointment_list).map { |appointments| OpenStruct.new(appointments) }
-      else
-        json_hash[:booked_appointment_collections].first[:booked_cc_appointments]
-                                                  .map { |appointments| OpenStruct.new(appointments) }
-      end
+      appointment_list = if type == 'va'
+                           json_hash.dig(:data, :appointment_list)
+                         else
+                           json_hash[:booked_appointment_collections].first[:booked_cc_appointments]
+                         end
+      return [] unless appointment_list
+
+      appointment_list.map { |appointments| OpenStruct.new(appointments) }
```

Comment:
this looks like it occurs after method returns.  is this just a ruby thing or is this not used?

Reply:
you could also do Array.wrap

since Array.wrap(nil) returns []

Reply:
@sjbrush ya `unless` is a rubyism to check that a statement returns false (e.g. instead of `if !something`), in this case it's a guard clause to return early if appointment_list is falsey